docs/reference/modules/node.asciidoc

```diff
@@ -89,19 +88,22 @@ be elected to become the master node by the <<modules-discovery,master election
 process>>.
 
 IMPORTANT: Master nodes must have access to the `data/` directory (just like
-`data` nodes) as this is where the cluster state is persisted between node restarts.
+`data` nodes) as this is where the cluster state is persisted between node 
+restarts.
 
-Indexing and searching your data is CPU-, memory-, and I/O-intensive work
-which can put pressure on a node's resources. To ensure that your master
-node is stable and not under pressure, it is a good idea in a bigger
-cluster to split the roles between dedicated master-eligible nodes and
-dedicated data nodes.
-
-While master nodes can also behave as <<coordinating-node,coordinating nodes>>
```

Comment:
Ah, I think we should still include this paragraph about not using dedicated master-eligible nodes as coordinating nodes, since "coordinating" isn't a role you can disable.

Reply:
@DaveCTurner 
I put the paragraph back (omitted its last sentence and merged the paragraph with the previous one) as you suggested. Thanks for spotting this.